src/tests/utils/compatibility-stories-factory.tsx

```diff
@@ -0,0 +1,24 @@
+import {RenderForm} from '@lib/renderer';
+import {PlayFunction} from '@storybook/csf/dist/story';
+import {Args, ComponentStory} from '@storybook/react';
+import React from 'react';
+// @ts-ignore
+import {Form} from 'react-formio';
+
+/**
+ * Generates a tuple of 2 stories, one for Form.io and one For RenderForm. Both stories receive
+ * `args` and `play` function.
+ */
+export const compatibilityStoriesFactory = (args: Args, play: PlayFunction) => {
+  const FormioStory = (args: Args) => <Form {...args} />;
+  FormioStory.args = args;
+  FormioStory.play = play;
+
+  const RenderFormStory: ComponentStory<typeof RenderForm> = args => <RenderForm {...args} />;
+  RenderFormStory.args = {
+    ...args,
+  };
```

Comment:
Nah this is a typo, noticed it already when resolving the dependency conflict. I don't think we need (deep) cloning here. IMO for this case this can be a plain reference.

Reply:
Actually on second though: I think the multiple branch might introduce some changes that might benefit from a clone.